features/other_b/skin.feature

```diff
@@ -433,3 +433,19 @@ Feature: creating and editing skins
     And I should see "only screen and (max-width: 42em), only screen and (max-width: 62em)"
   When I press "Use"
   Then the page should have a skin with the media query "only screen and (max-width: 42em), only screen and (max-width: 62em)"
+
+  Scenario: A user should be able to choose a temporary skin
+  Given basic skins
+    And the approved public skin "public skin" with css "#title { text-decoration: blink;}"
+   Then I am logged in as an admin
```

Comment:
Is it better to do that kind of thing or step though the interface, which will give us better coverage ?

I am happy to add the extra tests